Documentation.md

```diff
@@ -154,15 +154,15 @@ everything set up correctly and that you are able to listen for events.
           },
           region: "eu-central-1",         // REQUIRED for `CHAT`, optional otherwise
           softphone: {                    // optional
```

Comment:
maybe the comment on this should be "optional, default below apply if not provided"

Reply:
makes sense, will add it